src/fswAlgorithms/effectorInterfaces/solarArrayReference/_UnitTest/test_solarArrayReference.py

```diff
@@ -0,0 +1,227 @@
+    solarArrayWrap = unitTestSim.setModelDataWrap(solarArrayConfig)
+    solarArrayWrap.ModelTag = "solarArrayRotation"                                     # update python name of test module
+
+    # Add test module to runtime call list
+    unitTestSim.AddModelToTask(unitTaskName, solarArrayWrap, solarArrayConfig)
+
+    # Initialize the test module configuration data
+    solarArrayConfig.a1Hat_B = a1Hat_B
+    solarArrayConfig.a2Hat_B = a2Hat_B
+    solarArrayConfig.bodyFrame = bodyFrame
+
+    # Create input attitude navigation message
+    natAttInMsgData = messaging.NavAttMsgPayload()
+    natAttInMsgData.sigma_BN = sigma_BN
+    natAttInMsgData.vehSunPntBdy = rHat_SB_B
+    natAttInMsg = messaging.NavAttMsg().write(natAttInMsgData)
+    solarArrayConfig.attNavInMsg.subscribeTo(natAttInMsg)
+
+    # Create input attitude reference message
+    attRefInMsgData = messaging.AttRefMsgPayload()
+    attRefInMsgData.sigma_RN = sigma_RN
+    attRefInMsg = messaging.AttRefMsg().write(attRefInMsgData)
+    solarArrayConfig.attRefInMsg.subscribeTo(attRefInMsg)
+
+    # Create input spinning body message
+    hingedRigidBodyInMsgData = messaging.HingedRigidBodyMsgPayload()
+    hingedRigidBodyInMsgData.theta = thetaC
+    hingedRigidBodyInMsgData.thetaDot = thetaDotC
+    hingedRigidBodyInMsg = messaging.HingedRigidBodyMsg().write(hingedRigidBodyInMsgData)
+    solarArrayConfig.hingedRigidBodyInMsg.subscribeTo(hingedRigidBodyInMsg)
+
+    # Setup logging on the test module output message so that we get all the writes to it
+    dataLog = solarArrayConfig.hingedRigidBodyRefOutMsg.recorder()
+    unitTestSim.AddModelToTask(unitTaskName, dataLog)
+
+    # Need to call the self-init and cross-init methods
+    unitTestSim.InitializeSimulation()
+
+    # Set the simulation time.
+    # NOTE: the total simulation time may be longer than this value. The
+    # simulation is stopped at the next logging event on or after the
+    # simulation end time.
+    unitTestSim.ConfigureStopTime(macros.sec2nano(0.5))        # seconds to stop simulation
+
+    # Begin the simulation time run set above
+    unitTestSim.ExecuteSimulation()
+
+    if bodyFrame == 0:
+        thetaR = computeRotationAngle(sigma_RN, rHat_SB_N, a1Hat_B, a2Hat_B, thetaC)
+    else:
+        thetaR = computeRotationAngle(sigma_BN, rHat_SB_N, a1Hat_B, a2Hat_B, thetaC)
+    if thetaR-thetaC > np.pi:
+        thetaR -= np.pi
+    elif thetaR-thetaC < -np.pi:
+        thetaR += np.pi
+    # compare the module results to the truth values
+    if not unitTestSupport.isDoubleEqual(dataLog.theta[0], thetaR, accuracy):
+        testFailCount += 1
+        testMessages.append("FAILED: " + solarArrayWrap.ModelTag + "solarArrayRotation module failed unit test on thetaR for sigma_BN = [{},{},{}], sigma_RN = [{},{},{}] and bodyFrame = {} \n".format(sigma_BN[0],sigma_BN[1],sigma_BN[2],sigma_RN[0],sigma_RN[1],sigma_RN[2],bodyFrame))
```

Comment:
:large_orange_diamond: Reformat these two long lines. Drop each parameter onto a new line is an option.

Reply:
What do you mean here? Have each of those be multiple lines in the code? Or should it produce multiple lines when the error is being thrown?

Reply:
Fixed it.